app.js

```diff
@@ -0,0 +1,95 @@
+/*Problem statement.
+1.	Create a node.js command line program which reads operation(add, subtract, multiply, division) and values as arguments and returns the 
+appropriate output onto console as in below criteria
+
+a.	node app.js --operation addition 1 2 3 4  => 1+2+3+4 = 10
+b.	node app.js --operation multiply 1 2 3 4  => 1*2*3*4 = 24
+c.	node app.js --operation subtraction 2 1  => 2 - 1 = 1
+d.	node app.js --operation division 4 2  => 4/2 = 2
+
+2.	All the invalid cases need to be handled for all the operations
+3.	Include validations for unsupported operations, number of inputs etc.
+
+Note: Please note that addition and multiplication takes "n” values for inputs. Subtraction and division takes 2 values for input.
+ */
+
+// Solution:
+
+// Get the type of operation:
+let operation = process.argv.slice(3, 4);
+console.log(operation);
+
+// Get all the input numbers:
+let inputNumbers = process.argv.slice(4);
+console.log(inputNumbers);
+
+// Convert the above array of numbers,which is in 'String' format, to Number format:
+let inputs = [];
+for (let index = 0; index < inputNumbers.length; index++) {
+  inputs[index] = Number(inputNumbers[index]);
+}
+
+function performOperation(operation, numbers) {
```

Comment:
here, right now as an operation param is coming as an array?
Do we really need an operation is an array here?